Change <password> used in Egress Gateway for TLS Origination (#7504)
istio/istio.io: content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

    {{< /text >}}

1.  Generate the certificates for `nginx.example.com`.

>     Use any password with the following command by replacing password with your new password:

Reviewer: ```suggestion
    Run the following command, replacing `password` with your choice of password:
```
